rel/vars.config.in

```diff
@@ -67,7 +67,7 @@
 {http_api_client_endpoint, "8089"}.
 {s2s_use_starttls, "{s2s_use_starttls, optional}."}.
 {s2s_certfile, "{s2s_certfile, \"priv/ssl/fake_server.pem\"}."}.
-{sasl_mechanisms, "%%{sasl_mechanisms, [cyrsasl_scram, cyrsasl_plain]}."}.
+{sasl_mechanisms, "%%{sasl_mechanisms, [cyrsasl_scram_sha1, cyrsasl_plain]}."}.
```

Comment:
I think it'd be better to put `cyrsasl_scram_sha256` in the example here instead of `sha1`.

Reply:
In this and other examples I wanted to stick to SHA-1 due to the findings when trying to connect client application with SHA-256. If such comments aim to allow to have something quickly operational without digging into all the details I think it's best to provide examples that have higher changes of working out of box. For now, looking at the landscape of client applications I'd leave it with SHA-1.

Reply:
Makes sense :)